rfc/rfc004-verifiable-transactional-graph.md

```diff
@@ -54,21 +55,28 @@ In addition to required header parameters as specified in RFC7515 the following
 
 * **cty**: MUST contain the type of the transaction content indicating how to interpret it.
 * **crit** MUST contain the **sigt**, **ver**, **prevs** and **pal** headers.
+* **crit** MUST also contain the **lc** header if **ver** == 2.
```

Comment:
Should this be:
```suggestion
* **crit** MUST also contain the **lc** header if **ver** >= 2.
```
A future RFC can remove this requirement.

Reply:
No, since there's no version >= 2